src/components/AppMap.ts

```diff
@@ -804,6 +807,42 @@ export default class AppMap extends mixins(MixinUtil) {
     this.search();
   }
 
+  async addKorokType(name: string, query: string) {
+    let objs = await MapMgr.getInstance().getObjs(this.settings!.mapType,
+      this.settings!.mapName, query);
+    objs.forEach((obj: any) => {
+      obj.Translate = { X: obj.pos[0], Y: obj.pos[1], Z: obj.pos[2] };
+      obj.id = obj.korok_type;
```

Comment:
What does this do? Is there no other way of doing this? Not a big fan of overwriting a property with completely unrelated data...